cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -105,8 +105,9 @@ message Partition {
     string external_id = 2;
     //The human-friendly name for this partition
     string name = 3;
-    //The list of seismics that belong to this partition
-    repeated Seismic seismics = 4;
+    reserved 4;  // Used to be list of seismics.
```

Comment:
For compatibility with code I think we could leave the field in place but just note in a comment that it will be empty?

Reply:
This should still be compatible, I think? All protobuf fields being optional.

Reply:
Yeah, but it'll be a nuisance when upgrading the protobuf dependency - compile errors all around. =)

Reply:
:+1: